responsibleai_text/responsibleai_text/utils/genai_metrics/scripts/equivalence.py

```diff
@@ -48,6 +48,15 @@
 
 This rating value should always be an integer between 1 and 5. So the rating \
 produced should be 1 or 2 or 3 or 4 or 5.
+Some examples of valid responses are:
```

Comment:
this looks like the same repeating text in all of the prompts - maybe we can unify them by moving to a common constant in the common file and then importing and specifying that in all of these files?